### ISSUE TITLE
Updated cocoapods and now getting this error.

### ISSUE THREAD

**Issue by Wackybob** [AFNetworking/AFNetworking#2964]
At least 20 issues are shown.
I'm using Xcode 6.
```
AFNetworking Group
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:29:1: Unknown type name 'NS_ASSUME_NONNULL_BEGIN'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:36:1: Cannot combine with previous '(error)' declaration specifier
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:47:4: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:48:48: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:49:58: Expected ')'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:143:19: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:153:4: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:198:47: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:225:45: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:226:55: Expected ')'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:251:58: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:252:58: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:253:68: Expected ')'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:268:59: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:292:42: Expected ')'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:309:42: Expected ')'
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:320:36: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:356:32: Expected a type
/Pods/AFNetworking/AFNetworking/AFURLRequestSerialization.h:473:1: Unknown type name 'NS_ASSUME_NONNULL_END'
Too many errors emitted, stopping now
```

**Comment by istx25** [comment]
I've seen this before. Not sure what causes it.

**Comment by ahsbjunior** [comment]
Same thing here!

**Comment by kylef** [comment]
AFNetworking 2.6 introduces nullability annotations, these will depend on a modern version of Xcode 6 or Xcode 7. I would suggest you upgrade to Xcode 7 which supports these nullability annotations or use an older version of AFNetworking such as 2.5.4.

**Comment by kamalkumar1** [comment]
Fixed the issue by updating AFNetworking 2.6 to AFNetworking 2.5.4  for xcode 6.1

**Comment by istx25** [comment]
:+1:
